Gruntfile.js

```diff
@@ -0,0 +1,28 @@
+process.env.NODE_ENV = process.env.NODE_ENV || 'development';
+
+module.exports = function(grunt) {
+    grunt.initConfig({
+        pkg: grunt.file.readJSON('package.json'),
+        
+        jshint: {
+            all: {
+                src: ['Dable.js'],
+                options: {
+                    globalstrict: true,
+                    globals: {
+                        require: false,
+                        exports: false,
+                        module: false
+                    }
+                }
+            }
+        }
+    });
+
+    if (process.env.NODE_ENV === 'development') {
```

Comment:
safe to say this is a false positive. `process` is defined in the context of node/grunt.